deploy/pr/versions.tf

```diff
@@ -0,0 +1,24 @@
+terraform {
+  required_version = ">= 1.0.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">= 3.72"
+    }
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = ">= 2.10"
+    }
+    helm = {
+      source  = "hashicorp/helm"
+      version = ">= 2.4.1"
+    }
+  }
+
+  backend "s3" {
+    bucket = "terraform-ssp-github-actions-state"
+    key    = "pr/eks/terraform-main.tfstate"
+    region = "us-west-2"
+  }
+}
```

Comment:
remove this. see the above comment

Reply:
removed in https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/424/commits/33053a8c1d5785a00c0859fbcb24827faad6475d